CONTRIBUTING.md

```diff
@@ -16,6 +18,14 @@ Here are our guidlines for contributing:
 
 Please help us keep MLAPI open and inclusive. Read and follow our [Code of Conduct](CODE_OF_CONDUCT.md).
 
+## <a name="standards"></a> Coding Standards
+
+Your contributions must follow the [Unity C# Coding Standards](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/blob/master/CODING_STANDARDS.cs).
```

Comment:
Let's hoist this part from the conventions to here:

// |    - Our standard extends Microsoft's Framework Design Guidelines, which defines a number of rules not covered by this document. (see https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/).
// |    - This document inlcludes a subset of the most used rules as well as any additions and exceptions to the FDG.
// |    - If there is any disagreement between this file and the FDG, this file always wins. Exceptions to the FDG are marked with [FDG Exception]